pkg/k8shandler/cluster.go

```diff
@@ -69,14 +66,12 @@ func CreateOrUpdateElasticsearchCluster(dpl *v1alpha1.Elasticsearch, configMapNa
 		if err != nil {
 			return err
 		}
-	case action == v1alpha1.ElasticsearchActionStatusUpdateNeeded:
+	case action == v1alpha1.ElasticsearchActionNone || action == v1alpha1.ElasticsearchActionStatusUpdateNeeded:
+		// Scrape cluster health from elasticsearch every time
 		err = cState.UpdateStatus(dpl)
```

Comment:
since we need to do this every time, can we move this to outside of the `switch` statement?

Reply:
Yes, we can